SvadaTekst_Frida.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
+
+/*let tekstSvada = [
```

Comment:
Her har du utkommentert kode. 
For at koden vi skriver skal være ren, let leselig og vedlikeholdbar osv så godkjenner vi aldri pull requests som har utkommentert kode.